### PR TITLE
Update the community website regarding our open source project policy

### PR DESCRIPTION
**Pull request by s-pace**
We want to provde the best experience by prioritize open source project

### CODE REVIEW

**Comment by s-pace** [comment]
Always need the eye of our 💪 @codeharmonics

File: docs/source/documentation/4-docsearch-FAQ/2-not-an-open-source-project.html.md.erb [hunk 1]
```diff
@@ -12,6 +12,6 @@ You are not an open source project? Feel free to ask, open an issue on our repo
 
 By focusing on open source projects, we want to make our contribution to the open source community. We are learning every day thanks to this incredible way of learning. we want to help. We do ask a link to the project repo, e.g. gitlab, github or any other terms of use that outlines this peculiarity.
 
-## My project is already featured or uses a back up documentation generator tool?
+## My project is already featured or uses a *baked* up documentation-generator tool?
```
Comment (Haroenv) [inline docs/source/documentation/4-docsearch-FAQ/2-not-an-open-source-project.html.md.erb]: backed?

**Comment by vvo** [comment]
Gave feedback via a private gist, let me know

File: docs/source/documentation/4-docsearch-FAQ/2-not-an-open-source-project.html.md.erb (hunk 1 above)
Comment (codeharmonics) [reply docs/source/documentation/4-docsearch-FAQ/2-not-an-open-source-project.html.md.erb]: We discussed this title. I didn't understand it. Feel free to help us. cc @s-pace

Comment (Haroenv) [reply docs/source/documentation/4-docsearch-FAQ/2-not-an-open-source-project.html.md.erb]: TBH I don't really understand what the title / content is supposed to mean

Comment (s-pace) [reply docs/source/documentation/4-docsearch-FAQ/2-not-an-open-source-project.html.md.erb]: Yep thank you for sharing your concern @Haroenv 

I want to let people know that we will continue to provide DocSearch for website already using it and for new projects that use featured/supported documentation tools such as Docusaurus, gitbook, pkgdown, vuepress....

Comment (Haroenv) [reply docs/source/documentation/4-docsearch-FAQ/2-not-an-open-source-project.html.md.erb]: >*baked in*

that would be a better choice then IMO

File: docs/source/partials/join-form.haml [hunk 2]
```diff
@@ -1,21 +1,31 @@
...
+  .spacer16
+  %label.color-bunting.text-normal{:for => "open_source_policy"} URL to your open source policy:
+  .input-group
+    %span.input-group-addon
+      %i.icon.icon-link.color-bunting
+    %input.form-control{type: 'text', placeholder: 'https://MyOpenSourceTerms.com/', name: 'open_source_policy'}
+  %p.text-sm.color-east-bay.m-t-small
+    DocSearch is entirely handled for open-source projects.
+    Refer to 
+    %a{href: "https://community.algolia.com/docsearch/documentation/docsearch-FAQ/customize-configuration-file/", class:"clickable"} the projects we support
```
Comment (maxiloc) [inline docs/source/partials/join-form.haml]: I don't think the link is good.

**Comment by vvo** [comment]
image

Must be:
`URL of open-source repository`

---

image

Must be:

`DocSearch is [free for open-source projects](link to Projects we support), but you can also [configure and host it yourself](link to DocSearch own infra)`

**Pull request merged**